### PR TITLE
Fix Registered Issue for aws overlay of Jupyter-web-app

### PR DESCRIPTION
**Pull request by PatrickXYS**
**Which issue is resolved by this Pull Request:**
Fix Registered Issue for aws overlay of Jupyter-web-app

**Description of your changes:**
Use configmap with a different name rather than configmapGenerator, patch deployment to use overlay configmap.

**Checklist:**
- [ ] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate-changed-only`
    3. `make test`

### CODE REVIEW

File: jupyter/jupyter-web-app/overlays/aws/configMap.yaml [hunk 1]
```diff
@@ -0,0 +1,142 @@
+apiVersion: v1
```
Comment (Jeffwan) [inline jupyter/jupyter-web-app/overlays/aws/configMap.yaml]: change `configMap` to `config-map`, all rest files follows this convention.

File: jupyter/jupyter-web-app/overlays/aws/spawner_ui_config.yaml [hunk 2]
```diff
@@ -1,132 +1,9 @@
-# Configuration file for the Jupyter UI.
```
Comment (Jeffwan) [inline jupyter/jupyter-web-app/overlays/aws/spawner_ui_config.yaml:1]: Remove it if it's not being used.

File: jupyter/jupyter-web-app/overlays/aws/configMap.yaml [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    # If the 'readOnly' key is missing (defaults to 'false'), the respective option
+    # will be available for users to edit.
+    #
+    # Note that some values can be templated. Such values are the names of the
+    # Volumes as well as their StorageClass
+    spawnerFormDefaults:
+      image:
+        # The container Image for the user's Jupyter Notebook
+        # If readonly, this value must be a member of the list below
+        value: 527798164940.dkr.ecr.us-west-2.amazonaws.com/tensorflow-1.15.2-notebook-cpu:1.0.0
```
Comment (Jeffwan) [inline jupyter/jupyter-web-app/overlays/aws/configMap.yaml]: Let's have a # here. I know it's online, but this is actually dangerous sometimes.

File: jupyter/jupyter-web-app/overlays/aws/configMap.yaml [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+        #   - default-editor
+        value: []
+        readOnly: false
+kind: ConfigMap
+metadata:
+  labels:
+    app: jupyter-web-app
+    kustomize.component: jupyter-web-app
+  name: jupyter-web-app-jupyter-web-app-config-aws
+  namespace: kubeflow
```
Comment (Jeffwan) [inline jupyter/jupyter-web-app/overlays/aws/configMap.yaml]: Need a new line

File: jupyter/jupyter-web-app/overlays/aws/configMap.yaml (hunk 1 above)
Comment (PatrickXYS) [reply jupyter/jupyter-web-app/overlays/aws/configMap.yaml]: Fixed. And also change deployment-aws-patch

File: jupyter/jupyter-web-app/overlays/aws/spawner_ui_config.yaml (hunk 2 above)
Comment (PatrickXYS) [reply jupyter/jupyter-web-app/overlays/aws/spawner_ui_config.yaml:1]: Fixed.

File: jupyter/jupyter-web-app/overlays/aws/configMap.yaml (hunk 4 above)
Comment (PatrickXYS) [reply jupyter/jupyter-web-app/overlays/aws/configMap.yaml]: Fixed.

**Pull request closed without merging**